### PR TITLE
[TC-9079] Add simple webhook

### PR DESCRIPTION
**Pull request by marcmatt**
## Description

https://tradecloud.atlassian.net/browse/TC-9079

### Scope
This PR adds the "simple webhook", eg. the `simpleOrderEvent` field in the POST body

Review: https://app.gitbook.com/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/m7Tjr4MICDtLcSjMVj4B/

### Submitter pre-flight check

- [x] Review your documentation
- [x] Add labels `needs reviewing`
- [x] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

File: order/supplier/receive/README.md [hunk 1]
```diff
@@ -70,16 +75,27 @@ The `buyerAccountNumber` should be set on forehand in the Tradecloud connection
...
 
-`deliveryScheduleIncludingRequests`, `prices` and `pricesIncludingRequests` are only available in the new webhook, using the "Orders Webhook Integration" configuration in your company profile page, and also available in the `order-search` API when using polling.
+`deliveryScheduleIncludingRequests`, `prices` and `pricesIncludingRequests` are only available in the new webhook, using the "Orders Webhook Integration" configuration in your company profile page, and are also available in the `order-search` API when using polling.
 {% endhint %}
 
+## Current delivery line
+
+* `scheduledDelivery`: the current aggregated delivery line with logistics info, see [Simple Delivery Schedule](#simple-delivery-schedule) below.
+
+In case of the simple delivery schedule, there is no `scheduledDeliveryIncludingRequests` variant available. Please let [support](support.md) know when you need this field.
```
Comment (roy-tc) [inline order/supplier/receive/README.md:97]: Also, `scheduledDelivery` is actually based on `deliveryScheduleIncludingRequests`.
So it's `deliverySchedule` that is not present.

File: order/supplier/receive/README.md [hunk 2]
```diff
@@ -140,12 +156,36 @@ The `mergedItemDetails` will contain the original item details added by the buye
...
-### Requested planned delivery schedule
+## Native Delivery schedule
 
-`lines.deliverySchedule`: the requested delivery schedule by the buyer.
-* `position`: the optional position in the delivery schedule. Not to be confused with the `line.position`
-* `date`: the requested delivery date of this delivery schedule position. Date has ISO 8601 date `yyyy-MM-dd` format. See also [Standards](../../api/standards.md).
-* `quantity`: the requested quantity of this delivery schedule position. Quantity has a decimal `1234.56` format with any number of digits.
+`lines.deliverySchedule`: the current delivery schedule, either `Issued` or `Confirmed`.
+
+`lines.deliveryScheduleIncludingRequests`: the current planned delivery schedule, either `Issued`, `In Progress` (having an open `Proposal` or `Reopen Request`) or `Confirmed`. This field is only supported as native delivery schedule.
```
Comment (roy-tc) [inline order/supplier/receive/README.md:163]: This field is also used when using the Simple Delivery Schedule

**Review by roy-tc — COMMENTED** [review]
Some small adjustments still needed

File: api/delivery-schedule.md [hunk 3]
```diff
@@ -10,22 +10,25 @@ description: >-
...
-Some ERP systems like SAP work natively with delivery schedules.
+Some ERP systems like SAP work natively with multiple delivery lines per order line.
 Use the Tradecloud [**native** delivery schedule](#native-delivery-schedule) in this case.
 
-Other ERP systems can only work with one date and one quantity per order line.
+Other ERP systems can only work with only one delivery per order line.
 Use the Tradecloud [**simple** delivery schedule](#simple-delivery-schedule) in this case.
 
 ## Native delivery schedule
+To receive the native delivery schedule you must have the "Orders Webhook Integration" setting "My system supports" set to "**Multiple delivery lines per order line**".
```
[the remark this answers is not in the document]
Comment (marcmatt) [reply api/delivery-schedule.md:20]: Improved

File: order/buyer/receive/README.md [hunk 4]
```diff
@@ -54,8 +60,6 @@ First choose either the webhook API or the polling API to receive order response
 * `buyerLine`: the buyer part of the order line, see [Buyer line part](./#buyer-line-part).
 * `supplierLine`: the supplier part of the order line, see [Supplier line part](./#supplier-line-part).
 * `confirmedLine`: the order line as agreed between buyer and supplier, see [Confirmed line](./#confirmed-line).
-* `deliverySchedule`: the current aggregated delivery schedule with logistics info, see [Delivery Schedule](./#delivery-schedule) below.
```
[the remark this answers is not in the document]
Comment (marcmatt) [reply order/buyer/receive/README.md:57]: Yes, I moved it under sub header `Current delivery schedule` and `Current delivery line`

File: order/supplier/receive/README.md [hunk 5]
```diff
@@ -59,8 +66,6 @@ The `buyerAccountNumber` should be set on forehand in the Tradecloud connection
 * `buyerLine`: the buyer part of the order line, see [Buyer line part](./#buyer-line-part) below.
 * `supplierLine`: the supplier part of the order line, see [Supplier line part](./#supplier-line-part) below.
 * `confirmedLine`: the order line as agreed between buyer and supplier, see [Confirmed line](./#confirmed-line) below.
-* `deliverySchedule`: the actual aggregated delivery schedule with logistics info, see [Delivery Schedule](./#delivery-schedule) below.
```
[the remark this answers is not in the document]
Comment (marcmatt) [reply order/supplier/receive/README.md:62]: Yes, I moved it under sub header `Current delivery schedule` and `Current delivery line`

File: order/supplier/receive/README.md [hunk 6]
```diff
@@ -70,16 +75,27 @@ The `buyerAccountNumber` should be set on forehand in the Tradecloud connection
 * `mergedItemDetails`: detailed part information provided by both buyer and supplier, see [Item details](./#item-details).
 * `lastUpdatedAt`: is the latest date time the order line has been changed, usefull for polling.
 
+### Current delivery schedule
+
+* `deliverySchedule`: the current aggregated delivery schedule with logistics info, see [Native Delivery Schedule](#native-delivery-schedule) below.
+* `deliveryScheduleIncludingRequests`: the current aggregated delivery schedule including any open supplier or buyer requestso, see [Native Delivery Schedule](#native-delivery-schedule) below.
```
[the remark this answers is not in the document]
Comment (marcmatt) [reply order/supplier/receive/README.md:81]: Fixed

File: order/supplier/receive/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcmatt) [reply order/supplier/receive/README.md:97]: Fixed

Comment (marcmatt) [reply order/supplier/receive/README.md:97]: It says `scheduledDeliveryIncludingRequests` eg. `scheduledDelivery` + `IncludingRequests` variant

File: order/supplier/receive/README.md (hunk 2 above)
Comment (marcmatt) [reply order/supplier/receive/README.md:163]: There is no `deliveryScheduleIncludingRequests` or `scheduledDeliveryIncludingRequests` when using the simple delivery schedule.

File: order/buyer/receive/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (marcmatt) [reply order/buyer/receive/README.md:57]: Added

**Review by roy-tc — APPROVED** [review]
@marcmatt looks great!

**Pull request merged**